### PR TITLE
Added my plugin to backstage.io directory (dev-friends-day)

### PR DESCRIPTION
**Pull request by piatkiewicz**
Added [dev-friends-days](https://github.com/piatkiewicz/backstage-plugins/tree/main/dev-friends-days) reference to Plugins' Directory

This plugin offers `<DevFriendsDayCard />` which shows info if there is the Programmer, Tester or SysAdmin day today so you will know when to celebrate that special day with your team!

Example:
image

### CODE REVIEW

File: microsite/data/plugins/dev-friends-day.yaml
Comment (awanlin) [inline microsite/data/plugins/dev-friends-day.yaml:9]:
```suggestion
npmPackageName: 'backstage-dev-friends-days'
```

File: microsite/data/plugins/dev-friends-day.yaml
Comment (awanlin) [inline microsite/data/plugins/dev-friends-day.yaml:8]: This file will need to be included in this PR

File: microsite/data/plugins/dev-friends-day.yaml
Comment (awanlin) [inline microsite/data/plugins/dev-friends-day.yaml:6]:
```suggestion
description: Be informed if there is a Programmer, Tester or SysAdmin day today so you will know when to celebrate that special day with your team!
```

**Review by awanlin — CHANGES_REQUESTED** [review]
This seems like a fun plugin @piatkiewicz :)

I would say that perhaps it might be best to wait till this is further along based on this message in the docs: "Made for learning purposes mainly, very early stage of development and may contain weird changes in future."

Will leave final say to the Maintainers though. There are some things that need to be corrected either way.

**Review by tudi2d — DISMISSED** [review]
agree with andré's message above - let us know when you feel happy to move on with this.

**Comment by piatkiewicz** [comment]
I did some changes like extended description and added scope to the package name according to tips mentioned in backstage.io docs. I'd like to add it to plugins list on backstage.io already if it's possible. In future I'll add more dates to display and some configuration possibilities.

**Comment by piatkiewicz** [comment]
`<gently-unstaling>`

**Review by awanlin — APPROVED** [review]
Thanks for adding this fun plugin to the directory @piatkiewicz 🚀

**Comment by awanlin** [comment]
@tudi2d @benjdlambert can you give this a quick look, please? Good to go on my end! 🚀

**Review by tudi2d — APPROVED** [review]
:shipit:

**Pull request merged**
